cmd/init.go

```diff
@@ -586,6 +586,9 @@ func initNotifTemplates(path string, fs stuffbin.FileSystem, i *i18n.I18n, cs *c
 		"L": func() *i18n.I18n {
 			return i
 		},
+		"safe": func(s string) template.HTML {
```

Comment:
Let's rename this to `Safe` (uppercase S) to be consistent with the other exported functions.